code/game/objects/effects/landmarks.dm

```diff
@@ -45,7 +45,7 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark)
 	. = ..()
 	GLOB.start_landmarks_list += src
 	if(jobspawn_override)
-		LAZYADDASSOC(GLOB.jobspawn_overrides, name, src)
+		LAZYADDASSOCLIST(GLOB.jobspawn_overrides, name, src)
```

Comment:
That looks wrong, going off of the actual macros.

Reply:
The problem here is using LALZYADDASSOC you cannot actually remove the list entry anymore with LAZYREMOVEASSOC if you try to use the macro that way you will get a type missmatch runtime error for some reason if the object is wrapped inside a list when it gets applied this issue is solved so i used the LAZYADDASSOCLIST macro instead

Reply:
In this case this happens if the value entry is an object reference

Reply:
the second option here would be to completly disregard the macro and use this bit of code `testing["Test"] = null to null it, but leave the key, or
testing -= "Test" to remove both the key and value.` instead to remove the assoc value from the list / change the LAZYREMOVE macro iteslf to do that

Reply:
this is from coderbus by the way

Reply:
In its current state lazyremove asoc will not remove entries(object references) that have not been added while wrapped into a list however